packages/esm-patient-list-app/src/api/api-local.ts

```diff
@@ -0,0 +1,144 @@
+import { isOfflineUuid, offlineUuidPrefix } from '@openmrs/esm-framework';
+import Dexie, { Table } from 'dexie';
+import { PatientList, PatientListMember, PatientListUpdate, PatientListType, PatientListFilter } from '.';
```

Comment:
Imports from adjacent `index` are always problematic because they may be cyclic.

Reply:
(In this case I think it may only be for types? Remember an `enum` in TS is not a compile-time only construct, but will have runtime impact, too)